test/fixtures/poro.rb

```diff
@@ -33,6 +38,10 @@ def id
     @attributes[:id] || @attributes['id'] || object_id
   end
 
+  def to_param
+    id.to_s
+  end
```

Comment:
`to_param` and `to_model` are used in the for url generation.

`to_model` is called by `polymorphic_url` and it is expected to return an ActionModel compliant model. Without it, the `polymororphic_url`v method will error.
`to_param` is used to generate the value for the object in the url. Without it, I get a url like `http://test.host/tags/%23%3CTag:0xXXXXXX%3E/posts\`